playbooks/roles/deploy-osh/tasks/component-runtime-setup.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: "Generate suse override for {{ override.suse }}"
+  template:
+    src: "{{ override.suse }}.j2"
```

Comment:
Why `suse`? The meaning is really path/file name

Reply:
it's the suse-ification?

Reply:
It is, in general
-  but there's no non-suse user of socok8s and plenty of other pieces of code are SUSE specific
- I think variable name should explain it's meaning, and "suse" for a name of file just does not

Reply:
would `susefile` and `uservar` to replace `suse` and `user` be better?

Reply:
susefile sounds better, yes

Reply:
Done.